crates/bevy_render/src/render_resource/bind_group.rs

```diff
@@ -122,6 +125,19 @@ impl Deref for BindGroup {
 /// | `multisampled` = ...  | `true`, `false`                                                         | `false`              |
 /// | `visibility(...)`     | `all`, `none`, or a list-combination of `vertex`, `fragment`, `compute` | `vertex`, `fragment` |
 ///
+/// * `storage_texture(BINDING_INDEX, arguments)`
+///     * This field's [`Handle<Image>`](bevy_asset::Handle) will be used to look up the matching [`Texture`](crate::render_resource::Texture)
+///     GPU resource, which will be bound as a storage texture in shaders. The field will be assumed to implement [`Into<Option<Handle<Image>>>`]. In practice,
+///     most fields should be a [`Handle<Image>`](bevy_asset::Handle) or [`Option<Handle<Image>>`]. If the value of an [`Option<Handle<Image>>`] is
+///     [`None`], the [`FallbackImage`] resource will be used instead.
+///
+/// | Arguments              | Values                                                                                     | Default       |
+/// |------------------------|--------------------------------------------------------------------------------------------|---------------|
+/// | `dimension` = "..."    | `"1d"`, `"2d"`, `"2d_array"`, `"3d"`, `"cube"`, `"cube_array"`                             | `"2d"`        |
+/// | `image_format` = ...   | any member of [`TextureFormat`](bevy_render::render_resource::TextureFormat)               | `RgbaUniform` |
```

Comment:
It seems like bevy_render needs to be in scope. I think it would be easier to just list the valid options as strings instead. It's unfortunate that it can't be doctested easily but it's better than nothing.

Reply:
for `image_format` that would 73 options, which is why I did not take the conventional route of defining all the options up front. I try to see if I get get `bevy_render` in scope

Reply:
Oh, yeah in that case try to link it 😅

Reply:
should work now :)